python/tests/plugins/test_gen_design_doc_impl.py

```diff
@@ -14,13 +15,13 @@ def test_can_generate_design_doc_with_models(self):
             test_design_doc_file_name = None
             with open(test_model_file_name, "w") as arch_file:
                 arch_file.write(TEST_MODEL)
-
+    
```

Comment:
We need to remove the extra spaces here. You may have a Neovim setting that allows removing space at the end of lines - if that's available, please set that (at least for this project). The linter should catch this, too, if I'm not mistaken.

Reply:
I believe that you're correct; flake8 should catch this.

Reply:
Just pushed some changes through let me know if there is anything else